subgraphs/radiant-capital/README.md

```diff
@@ -0,0 +1,171 @@
+# RADIANT
+## Business Summary
+Radiant is a cross chain lending protocol that is forked from Aave and Geist. It’s revenue split and incentive scheme are modelled after Geist Finance on Fantom chain.
+
+Radiant launched v1 on Arbitrum now. It’s working on v2, which allows for full cross chain borrowing & lending on assets such as BTC, ETH and USDC.
+
+The business of the protocol can be divided and summarized as follows:
+
+### Lending and Borrowing
+
+- Depositors can lend supply assets to supply market and in exchange, they receive rTokens, which generate yield (interest rate payments from borrowers).
+- Borrowers post collateral and can borrow on interest rates that are algorithmically determined based on supply & demand for each asset.
+- Liquidators liquidate undercollateralized positions for liquidation penalty.
+
+### 1-Click Looping
+
+- Looping is automating the deposit and borrow the same asset and repeat the cycle several times in one transaction. This is also known as folding in DeFi. In Radiant Finance, a user can loop up to 5x leverage. 
+
+-  Loop function allows users to increase their RDNT token rewards and earn yield on a larger collateral value 
+
+### Vesting and Locking RDNT
+
+#### Vesting RDNT
+- Users vest their RDNT tokens to receive 50% of all fee-generated (borrowing interest paid) while actively vesting. 
+
+- Users can withdraw their vested tokens for a 50% penalty on the RNDT rewards.
+
+- The 50% penalty of RDNT rewards are distributed to users who lock their RDNT tokens
+
+##### Locking RDNT
+- Users who lock their RDNT tokens receive 50% of fee generated as well. 
+
+- They will also receive additional income from RDNT penalty fees from all positions who exit their vests early. 
+
+- Users who lock their tokens can unlock but cannot fully withdraw them.
+
+### Pool2 
+- Liquidity providers of RDNT/WETH pair on SushiSwap can stake their LP tokens in Radiant to receive rewards in the form of RDNT tokens 
+- This pool aims to enable deeper liquidity for the protocol to secure the longevity and health of Radiant platform
+
+## Useful Links
+Protocol:
+- https://app.radiant.capital/#/markets 
+
+Docs:
+- https://docs.radiant.capital/docs/ 
+
+Smart contracts:
+- https://docs.radiant.capital/docs/contracts-and-security/contracts 
+
+Discord:
+- https://discord.com/invite/radiantcapital
+
+## User Metrics
+The user actions of Radiant include:
+
+### Lending and Borrowing
+#### Lending
+- Deposit any of the five supported assets such as WBTC, WETH, USDT, USDC, & DAI in exchange of rTokens
+- Claim RDNT rewards from the deposited assets
+
+#### Borrowing
+- Borrow after depositing assets as collaterals
+- Repay borrowed funds
+- Claim RNDT rewards from the borrowed assets
+
+#### Liquidation
+- Liquidate undercollateralized positions/loans
+
+### 1-Click Looping
+- Automate and repeat the process of borrow and deposit cycle multiple times by looping to earn greater yield
```

Comment:
Does this count as one user action? or one per underlying?
If the user loops 5 times, will it count 5 deposits + 5 borrows + 10 transactions? or 5 deposits + 5 borrows + 1 transaction?

Reply:
I suggest we judge basing on if there's a separate function for this in the smart contract. If not a smart contract function but just a UI feature, this is not different from a user doing this loop manually.

Reply:
Hmm, it is an additional contract, separate from the main aave forked ones. A single call to this contract results in 5 + 5 calls to the lending pool. It all happens in a single EVM transaction.

If we observe the main contracts directly it will seem as individual actions. But from the UI it will be a single transaction or action.

Reply:
Ok noted. I guess it's a judgement call for Messari protocol specialist then.

Reply:
@ishraq8 what do you think?

Reply:
Here is my logic. In AAVE we consider manual looping as 5+5, in this it is automated through a separate contract (5+5). I am thinking keeping it 5+5 to keep the counting consistent between AAVE and forks like this. It also makes it easier to compare to AAVE. We may want to consider it one transaction on the front end depending how we digest this in the future.